util/logging/retrieve-cluster-logs.py

```diff
@@ -30,18 +30,16 @@
 logging.basicConfig(level=logging.INFO)
 LOGGER = logging.getLogger(__file__)
 
+DEFAULT_BUCKET_PREFIX_FORMAT = "{{cluster_name}}-logs-{timestamp}".format(timestamp=datetime.now().timestamp())
```

Comment:
Interesting use of python string formatting 👍

Reply:
is this compatible with Python 2.7. I know this script is not part of our CLI but it would be nice to guarantee same python compatibility if possible

Reply:
Yes it is.